package.json

```diff
@@ -44,13 +44,16 @@
     "packages/xliff-to-json-converter",
     "packages/nimble-tokens",
     "packages/nimble-components",
+    "packages/ok-components",
```

Comment:
oh nice, this build order is backwards, if okay was actually using spright this would be broken :P